src/Simplex/Chat.hs

```diff
@@ -4413,17 +4426,17 @@ chatCommandP =
       "/_remove #" *> (APIRemoveMember <$> A.decimal <* A.space <*> A.decimal),
       "/_leave #" *> (APILeaveGroup <$> A.decimal),
       "/_members #" *> (APIListMembers <$> A.decimal),
-      -- /smp_servers is deprecated, use /smp and /_smp
-      "/smp_servers default" $> SetUserSMPServers (SMPServersConfig []),
-      "/smp_servers " *> (SetUserSMPServers . SMPServersConfig . map toServerCfg <$> smpServersP),
-      "/smp_servers" $> GetUserSMPServers,
-      "/smp default" $> SetUserSMPServers (SMPServersConfig []),
-      "/_smp test " *> (APITestSMPServer <$> A.decimal <* A.space <*> strP),
-      "/smp test " *> (TestSMPServer <$> strP),
-      "/_smp " *> (APISetUserSMPServers <$> A.decimal <* A.space <*> jsonP),
-      "/smp " *> (SetUserSMPServers . SMPServersConfig . map toServerCfg <$> smpServersP),
-      "/_smp " *> (APIGetUserSMPServers <$> A.decimal),
-      "/smp" $> GetUserSMPServers,
+      "/_server test " *> (APITestProtoServer <$> A.decimal <* A.space <*> strP),
+      "/smp test " *> (TestProtoServer . AProtoServerWithAuth SPSMP <$> strP),
+      "/xftp test " *> (TestProtoServer . AProtoServerWithAuth SPXFTP <$> strP),
+      "/_servers " *> (APISetUserProtoServers <$> A.decimal <* A.space <*> srvCfgP),
+      "/smp " *> (SetUserProtoServers . APSC SPSMP . ProtoServersConfig . map toServerCfg <$> protocolServersP),
+      "/smp default" $> SetUserProtoServers (APSC SPSMP $ ProtoServersConfig []),
+      "/xftp " *> (SetUserProtoServers . APSC SPXFTP . ProtoServersConfig . map toServerCfg <$> protocolServersP),
+      "/xftp default" $> SetUserProtoServers (APSC SPXFTP $ ProtoServersConfig []),
+      "/_servers " *> (APIGetUserProtoServers <$> A.decimal <* A.space <*> strP),
+      "/smp" $> GetUserProtoServers (AProtocolType SPSMP),
+      "/xftp" $> GetUserProtoServers (AProtocolType SPXFTP),
```

Comment:
/xftp is already used for APISetXFTPConfig

Reply:
with parameters, so should be ok